api/PointerEvent.json

```
@@ -867,6 +867,57 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "fractional_coordinates": {
+          "__compat": {
+            "description": "Fractional coordinates for <code>mouse</code>.",
+            "support": {
+              "chrome": {
+                "version_added": "64"
+              },
+              "chrome_android": {
+                "version_added": "64"
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": null
+              },
+              "firefox_android": {
+                "version_added": null
+              },
+              "ie": {
+                "version_added": null
+              },
+              "opera": {
+                "version_added": null
```

The status page entry you linked to says "51" here. Probably safe to include here?

```suggestion
                "version_added": "51"
```